src/main/java/com/bunq/sdk/exception/BadRequestException.java

```diff
@@ -0,0 +1,8 @@
+package com.bunq.sdk.exception;
+
+public class BadRequestException extends BunqError {
+
+  public BadRequestException(String message, int responseCode) {
+    super(message, responseCode);
+  }
+}
```

Comment:
One newline before the closing brace of class please :)

Reply:
Oh yea forgot to do this indeed 😁